package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "api-weatherapp-js",
+  "version": "1.0.0",
+  "description": "Create a weather app using JavaScript and weather API.",
+  "private": true,
+  "dependencies": {
+    "css-loader": "^4.2.1",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "lodash": "^4.17.20",
+    "style-loader": "^1.2.1",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.8.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
```

Comment:
**REQUIRED**
Let's add these packages as **devDependancies** since we only need them in the development environment. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).